README.md

```diff
@@ -205,3 +205,54 @@ Add new dependencies to `Pipfile`, run:
     pipenv install <name>
 
 and then commit the changes to `Pipfile` and `Pipfile.lock`.
+
+
+
+
+### Configuring your local development environment
+#### Prerequisites
+This tutorial assumes you have the following dependencies already installed.
+* Git
+* Python
+* PostgreSQL 
+
+1. Clone the git repo
+```sh
+git clone git@github.com:seattleflu/id3c.git
+```
+
+2. To fill the `seattleflu` database with the backup db copy, run 
+`pg_restore --data-only --single-transaction --schema receiving --table enrollment --dbname seattleflu seattleflu-production-<date>.pgdb`
+
+3. To restore the presence/absence data on your local dev instance: 
+`pg_restore --dbname seattleflu presence_absence.pgdb`
```

Comment:
Steps 2 and 3 here are useful to have in a personal notebook, but aren't generally-applicable to the README's intended audience. Most folks don't have a `seattleflu-production-<date>.pgdb` file or `presence_absence.pgdb` file.

That said, it _would_ be good to have a section or another file (`DEVELOPMENT.md`?) on common development tasks. We could take those two specific examples and turn them into more generally-applicable examples of using both `pg_dump` and `pg_restore` for various use cases (dump+load a specific table, dump+load the entire database, etc).

Reply:
Should a `DEVELOPMENT.md` file live outside of this repo? 
I'm maintaining an Ubuntu setup guide on the `blab/wiki`. Maybe a development guide could live over there, too? 

I'm also happy to generalize these instructions for a broader audience in addition to our specific purposes.

Reply:
I was thinking of `DEVELOPMENT.md` as another file in the root of this repo that was a list of typical tasks around id3c development, something like:

```md
# Development of ID3C

## Common tasks

### Dumping and loading a specific table from production into your local dev database

…

### Dumping and loading the production database into your local dev database

…

### Resetting ETL processing logs for tables in the receiving schema

…
```

and so forth.